src/main/webapp/app/pages/annotationPage/AnnotationPage.tsx

```diff
@@ -92,12 +93,60 @@ export type IAnnotationPage = {
   ) => void;
 };
 
+const MutationEffectInformation: React.FunctionComponent<{
+  show: boolean;
+  mutationEffect: string | undefined;
+  onClick: () => void;
+  className?: string;
+}> = props => {
+  return (
+    <div className={props.className}>
+      <div onClick={() => props.onClick()}>
+        <i>{`${props.show ? 'Hide' : 'Show'} mutation effect description`}</i>{' '}
+        <ShowHideToggleIcon show={props.show} onToggle={() => {}} />
+      </div>
+      {props.show ? (
```

Comment:
If `props.mutationEffect` is empty, let's skip rendering MutationEffectInformation